core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -199,8 +199,10 @@ private[query] class EventsByTagPublisher(
         context.system.scheduler.scheduleOnce(settings.eventualConsistencyDelay, self, Continue)
   }
 
-  def timeToLookForDelayed: Boolean =
-    strictBySeqNumber && !backtracking && lookForDelayedDeadline.isOverdue() && !isTimeBucketBeforeToday()
+  def timeToLookForDelayed: Boolean = {
+    strictBySeqNumber && toOffset.isEmpty && !backtracking &&
+      lookForDelayedDeadline.isOverdue() && !isTimeBucketBeforeToday()
+  }
```

Comment:
this would probably be the only thing need to fix the reported issue with `currentEventsByTag`, but I wanted to understand the root cause and fix that also since that could be used in the live query (so the issue is fixed kind of twice)